pkg_resources/_vendor/packaging/requirements.py

```diff
@@ -90,8 +90,12 @@ def __init__(self, requirement_string):
             req = REQUIREMENT.parseString(requirement_string)
         except ParseException as e:
             raise InvalidRequirement(
-                "Invalid requirement, parse error at \"{0!r}\"".format(
-                    requirement_string[e.loc:e.loc + 8]))
+                "Invalid requirement for \"{0}\", "
```

Comment:
You can't make changes in _vendor - those changes have to first be made upstream, released, then update the vendored package.